network/network.go

```diff
@@ -138,6 +144,45 @@ func (n *Network) Configure(config core.ServerConfig) error {
 	return nil
 }
 
+// TODO: Untangle from v1 and move to ConnectionManager/ManagedConnection
```

Comment:
Yes. I'm not completely satisfied with this being here since I'd rather have it in `grpc` but then it's all about cyclic dependencies again, unless I feed it with all individual arguments.

I refactored it to move some of the `grpc` internals to `grpc/config.go`.